package.json

```diff
@@ -17,8 +17,16 @@
     "build:umd": "cross-env BABEL_ENV=commonjs webpack",
     "build:umd:min": "cross-env BABEL_ENV=commonjs NODE_ENV=production webpack",
     "build:watch": "npm run build:commonjs -- -watch",
+    "precommit": "lint-staged",
     "prepublish": "npm run clean && npm run build"
   },
+  "lint-staged": {
```

Comment:
For now, I'm living with doing `prettier --list-different` in my pre-commit hook to fail the commit on unformatted files and then adding a `format` script to actually do the formatting. Not ideal, but better than surprising people IMO.